index.adoc

```diff
@@ -602,6 +602,38 @@ MUST NOT perform type validation on any value in the array before submitting the
 
 Use of this member is OPTIONAL.
 
+A non-normative example
+
+[source,json]
+----
+{
+  "href": "https://example.io/persons", "rel": ["create-form"], "method": "POST",
+  "value": [
+    {
+      "name": "friends", "type": "object", "form": {
+        "type": "set", "itemtype": "object",
+        "value": [
+          { "name": "firstName" },
+          { "name": "email" }
+        ]
+      }
+    }
+  ]
+}
+----
```

Comment:
Hi @lhazlewood .

Thanks for your feedback!
I think the change to eform and etype is making the specification easier to read - one specific word for each concept. I think introducing two new members is a resonable price for readability.